packages/@expo/cli/src/export/createMetadataJson.ts

```diff
@@ -38,10 +38,11 @@ export function createMetadataJson({
             bundle: path.join('bundles', fileNames[platform]!),
             // Collect all of the assets and convert them to the serial format.
             assets: bundle.assets
+              .filter((asset: any) => !asset.embedded)
               .map(
-                (asset) =>
+                (asset: any) =>
                   // Each asset has multiple hashes which we convert and then flatten.
-                  asset.fileHashes?.map((hash) => ({
+                  asset.fileHashes?.map((hash: string) => ({
```

Comment:
optional: after adding types, this one can be inferred

```suggestion
                  asset.fileHashes?.map((hash) => ({
```